### PR TITLE
deps: update various deps to fix security warnings

### PR DESCRIPTION
**Pull request by BethGriggs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,19 +9,19 @@
     "express": "^4.14.1"
   },
   "devDependencies": {
-    "codecov": "^2.1.0",
+    "codecov": "^3.6.1",
     "compression": "^1.6.2",
     "cors": "^2.8.1",
     "eslint": "^3.14.1",
```
Comment (richardlau) [inline package.json]: https://github.com/CloudNativeJS/appmetrics-prometheus/network/alerts also flags eslint:
image

Comment (BethGriggs) [reply package.json]: Force pushed this in, upgraded to v4.18.2 and `eslint` still seems to pass.

**Review by richardlau — APPROVED** [review]
CI is going to fail with Node.js 13 until we get a version of `appmetrics` containing https://github.com/RuntimeTools/appmetrics/pull/619.

**Comment by mattcolegate** [comment]
Appmetrics 5.1.1 has been released wtih Node 13 support. Please re-do the Travis checks (I haven't permission)

**Pull request merged**